docs/rules/no-done-callback.md

```diff
@@ -1,5 +1,15 @@
 # Avoid using a callback in asynchronous tests and hooks (`no-done-callback`)
 
+<!-- RULE_NOTICE -- Generated by `yarn tools:regenerate-docs` -->
+
+<!-- prettier-ignore -->
+💼 This rule is enabled in the following [configs](https://github.com/jest-community/eslint-plugin-jest#shareable-configurations): `all`, `recommended`.
+
+<!-- prettier-ignore -->
+💡 This rule is manually fixable by editor [suggestions](https://eslint.org/docs/developer-guide/working-with-rules#providing-suggestions).
```

Comment:
nit: what do you think of "This rule provides suggestions that can be applied manually"? That feels a bit more grammatically natural to me

Reply:
That's fine, updated.